src/UserALEWebExtension/options.ts

```diff
@@ -76,7 +75,46 @@ function getConfig() {
     (document.getElementById("filter") as HTMLInputElement).value =
       payload.pluginConfig.urlWhitelist;
   });
+
+  (document.getElementById("optionsForm") as HTMLFormElement).addEventListener(
+    "submit",
+    setConfig,
+  );
+  (document.getElementById("issueForm") as HTMLFormElement).addEventListener(
+    "submit",
+    reportIssue,
+  );
+}
+
+function reportIssue() {
+  browser.runtime.sendMessage({
+    type: MessageTypes.ISSUE_REPORT,
+    payload: {
+      issueType: (
+        document.querySelector(
+          'input[name="issueType"]:checked',
+        ) as HTMLButtonElement
+      ).value,
+      issueDescription: (
+        document.getElementById("issueDescription") as HTMLTextAreaElement
+      ).value,
+      type: "issue",
```

Comment:
Do we want to add these as additional fields or put them in the "details" field? If I recall, you added a discussion ticket for us to standardize our usage of details based on some conventions.

Right now it contains additional, action-specific contextual information for certain event types. Feels like a natural place to include these fields.

Reply:
I'll move to details.